darts/models/forecasting/multivariate_forecasting_model_wrapper.py

```diff
@@ -0,0 +1,140 @@
+"""
+Multivariate forecasting model wrapper
+-------------------------
+
+A wrapper around local forecasting models which allows forecasting multiple components
+"""
+from typing import List, Optional, Tuple
+
+from darts.logging import get_logger
+from darts.models.forecasting.forecasting_model import (
+    FutureCovariatesLocalForecastingModel,
+    LocalForecastingModel,
+    TransferableFutureCovariatesLocalForecastingModel,
+)
+from darts.timeseries import TimeSeries, concatenate, split_multivariate
+from darts.utils.utils import seq2series
+
+logger = get_logger(__name__)
+
+
+class MultivariateForecastingModelWrapper(FutureCovariatesLocalForecastingModel):
+    def __init__(self, model: LocalForecastingModel):
+        """
+        Wrapper around LocalForecastingModel allowing it to predict multivariate TimeSeries.
+
+        A copy of the provided model will be trained on each of the components of the provided series separately.
+        Parameters
+        ----------
+        model
+            Model used to predict individual components
+        """
+        super().__init__()
+
+        self.model: LocalForecastingModel = model
+        self._trained_models: List[LocalForecastingModel] = []
+
+    def _fit(self, series: TimeSeries, future_covariates: Optional[TimeSeries] = None):
+        super()._fit(series, future_covariates)
+        self._trained_models = []
+
+        series = seq2series(series)
+        components = split_multivariate(series)
+        if self.supports_future_covariates:
+            self._trained_models = [
+                self.model.untrained_model().fit(c, future_covariates)
+                for c in components
+            ]
+        else:
+            self._trained_models = [
+                self.model.untrained_model().fit(c) for c in components
+            ]
```

Comment:
To make this more intuitive, I would rather loop on the components on the series directly and use an in-line if-else:

```suggestion
        for comp in series.components:
            self._trained_models.append(
                self.model.untrained_model().fit(
                    series=series[comp],
                    future_covariates=future_covariates if self.supports_future_covariates else None
                )
            )
```